pkgs/applications/networking/instant-messengers/discord/openasar.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, nodejs-16_x, bash, nodePackages }:
```

Comment:
I think it is better to use `nodejs` here instead. It should always point to the most recent LTS version.

In case it doesn't build for some reason you can override it at the call site, but it shouldn't be necessary since the currently `nodejs` points to `nodejs-16_x` anyway.